phoenicis-dist/src/flatpak/org.phoenicis.javafx.json

```diff
@@ -0,0 +1,123 @@
+{
+  "app-id": "org.phoenicis.javafx",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk11"
+  ],
+  "add-extensions": {
+    "org.freedesktop.Platform.Compat.i386": {
+      "directory": "lib/i386-linux-gnu",
+      "version": "18.08"
+    }
+  },
+  "command": "phoenicis-javafx",
+  "finish-args": [
+    "--allow=multiarch",
+    "--share=ipc",
+    "--socket=x11",
+    "--share=network",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "setup-compat-i386",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/lib/i386-linux-gnu",
+        "install -Dm644 -t /app/etc ld.so.conf"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "ld.so.conf"
+        }
+      ]
+    },
+    {
+      "name": "openjdk",
+      "buildsystem": "simple",
+      "build-commands": [
+        "/usr/lib/sdk/openjdk11/install.sh"
+      ]
+    },
+    {
+      "name": "openjfx",
```

Comment:
Is it still required to install OpenJFX separately?
Starting with Java 11 it should be possible to install JavaFX/OpenJFX via maven.

Reply:
Doesn't work somehow. I get:
```
Error occurred during initialization of boot layer
java.lang.module.FindException: Module javafx.controls not found
```

Reply:
I think this indicates a problem with the `java` command and/or the project setup for Java 11.
Have you only removed the 
```
{
      "name": "openjfx",
      ...
}
```
block?
Have you also removed the `-p /app/javafx-sdk-11/lib` argument to the `java` command?

Reply:
The problem is: I have only JDK10 here (Ubuntu 18.04). If you check #1408, the maven dependencies do not seem to be enough (according to the tests of @ImperatorS79).